docs/DeveloperGuide.md

```diff
@@ -1,37 +1,1040 @@
+
+(Step 10) Upon the successful execution of the `DayList#printSchedule()` method, the plan scheduled on each of the day
+will be display on the console to the user. An expected outcome of the `schedule /list` command would be:
+
+```
+----------------------------------------------------------------------
+
+                         WORKOUT SCHEDULE
+----------------------------------------------------------------------
+     Day       |            Plan Name
+----------------------------------------------------------------------
+      Monday   |              arms                          
+     Tuesday   |            rest day                      
+   Wednesday   |            leg day                      
+    Thursday   |            rest day                      
+      Friday   |            rest day                      
+    Saturday   |            leg day                      
+      Sunday   |            rest day                      
+
+----------------------------------------------------------------------
+```
+
+By default, if no plan is being scheduled for any of the day, the day is to be considered as a rest day for the user.
+
+---
+#### Clear Schedule For A Day
+A summary of the general procedure of clearing a plan scheduled for a paricular day of the schedule in WerkIt! is as follows:
+1. User enters the command `schedule /clear <day number>`.
+2. The application will locate the index in the DayList which stores the corresponding `Day` object.
+   This `Day` object will then be deleted from the DayList. For example, if `schedule /clear 1` command is entered,
+   the index where the `Day` object storing information of the plan scheduled for Monday would be store in index 0,
+   (day number -1), of the DayList.
+3. DayList[day number - 1] would become null after the command is successfully being executed.
+4. The success response is printed to the user through the terminal.
+5. The `schedule.txt` will also be rewritten to reflect the changes. 
+
+The following sequence illustrates how the `schedule /clear` command works in greater detail:
+> To simplify the sequence diagram, some method invocations that deemed to be trivial
+> have been removed from the sequence diagram. Reference frames will be elaborated further
+> down this section.
+
+![ClearSchedule](uml/sequenceDiagrams/schedule/images/clearSchedule.png) 
+<br><br>
+
+(Steps 1 to 3) The program waits for the user's input, which in this case,
+is the `schedule /clear <day number>` command. An example of a valid command would be `schedule \clear 1`
+Once the command is entered, the UI class will return the user input in a String object to the WerkIt object.
+
+(Steps 4 to 5) After the user input is received, the WerkIt object will call the `Parser#parseUserInput()` method
+to parse the user input. A return ScheduleCommand object is obtained from the method.
+This ScheduleCommand object is upcasted to a Command object on return to the WerkIt object.
```

Comment:
Steps here are not bolded in some parts, but are bolded in parts such as Creating a Plan. Try to standardise!